### ISSUE TITLE
`No protocol method Unmaskable.unmask defined for type null` if tap several times on Sign Transaction 

### ISSUE THREAD

**Issue by annadanchenko** [status-im/status-react#4905]
### Description

*Type*: Bug

*Summary*: `No protocol method Unmaskable.unmask defined for type null` error popup is shown after tapping several times on Sign transaction button if Send transaction screen is opened from DApp

Note: on iOS after getting such error nothing happens if user tries to tap on other DApp or app buttons, so need to restart app to proceed working with it

#### Expected behavior
Can tap only once on Sign transaction button, after this button is disabled

#### Actual behavior
`No protocol method Unmaskable.unmask defined for type null` error popup is shown after tapping several times on Sign transaction button 
### Reproduction
iOS: https://app.testfairy.com/projects/4803590-status/builds/8502922/sessions/4394588605/?accessToken=vCsHkQZFMJOimRXFJEi2Hka01OE
- Open Status, switch to ropsten network
- open Test Dapp -> Assets -> request STT. As a result, Send transaction screen is opened
- provide correct password and quickly tap several times on Sign transaction button 
- check errors if any shown after or before Got it screen

### Additional Information
* Status version: release
* Operating System: found on Android and iOS

**Comment by annadanchenko** [comment]
duplicate of #4858
